### ISSUE TITLE
Request for translation branches and sites

### ISSUE THREAD

**Issue by guoyunhe** [openSUSE/news-o-o#27]
The plan is to create branches:

- lang/de
- lang/fr
- lang/ja
- lang/pt
- lang/zh
- ...

And they will be deployed to:

- news.opensuse.org/de/
- news.opensuse.org/fr/
- news.opensuse.org/ja/
- news.opensuse.org/pt/
- news.opensuse.org/zh/

How contributors participate translations:

1. Fork the repo
2. Checkout the lang branch
3. Merge master branch to get latest posts
4. Translate latest posts
5. Commit and create PR (to the lang branch)
6. Review, accept and deploy the lang branch

**Comment by hellcp** [comment]
Sure, but I would be curious what amount of interest there is in translations, and who would be responsible for reviewing contents of the PRs to different branches (and it would be best if the translator and the reviewer were two different people). I think that should be figured out first and then we can figure out which languages require branches.

**Comment by guoyunhe** [comment]
Yes. It can be created on request. The Chinese community has been discussing about it for some time. Maybe not every article, but we do want to translate articles of some important events, like oSC, Leap milestones. If the Chinese site is going well, then other community may also join.

**Comment by vinzv** [comment]
That all sounds a bit too much work "just" to get translations ready. Let's stick with original plan by @guoyunhe.

**Comment by hellcp** [comment]
I would argue it's easier overall, it's not a terrible idea

**Comment by vinzv** [comment]
Okay, sounded more troublesome to me. Don't let yourself get kept back from work. :-)

**Comment by guoyunhe** [comment]
the original branches solution has an advantage: local communities can publish local news article, like a local meet up, release parties.

the weblate solution is a bit hard for scripting.

the branch solution is a bit hard for making and reviewing PRs.

**Comment by vinzv** [comment]
> the branch solution is a bit hard for making and reviewing PRs. for every contributors.

Yes, but luckily they can do all of that even via the web interface here on GH. That makes it easier for not so tech-savvy writers.

Btw: I'd go with `news.opensuse.org/lang-code/` to make things more accessible.

**Comment by hellcp** [comment]
> Btw: I'd go with `news.opensuse.org/lang-code/` to make things more accessible.

Allows us to avoid having to generate a new certificate for *.news.opensuse.org ;)

**Comment by alexandrevicenzi** [comment]
Hi, What is the status of this? We would like to have a pt-BR version of news.

Hugo has very good support for translations, everything is build-in into it. Migrating from Jekyll won't be that hard. Hugo is still similar and will require some coding knowledge to make it work, but a lot of things can be done with GH Actions and GH Pages.

On the other hand, Hugo has good integration with Netlify, and this allows users without coding skills to create, update, and review articles.

I would be willing to give it a try, along with some folks from Brazil we could set up and migrate everything.

**Comment by hellcp** [comment]
Currently the effort here mostly moved to [newscom](https://github.com/openSUSE/newscom), so you should probably ask there. We do have a plugin in jekyll that enables us to do translations though, take a look at https://get.opensuse.org as an example
